framework/source/class/qx/ui/embed/Iframe.js

```diff
@@ -69,12 +69,25 @@ qx.Class.define("qx.ui.embed.Iframe",
 
     if ((qx.core.Environment.get("engine.name") == "gecko"))
     {
-      this.addListenerOnce("appear", function(e)
+      this.addListenerOnce("appear", function ()
       {
-        var element = this.getContentElement().getDomElement();
-        qx.bom.Event.addNativeListener(element, "DOMNodeInserted", this._onDOMNodeInserted);
-      });
-      this._onDOMNodeInserted = qx.lang.Function.listener(this._syncSourceAfterDOMMove, this);
+        // Mutation record check callback
+        var isDOMNodeInserted = function (mutationRecord) {
+          return mutationRecord.addedNodes;
+        };
+
+        var observer = new MutationObserver(function (mutationRecords)
```

Comment:
@oetiker : Independent of my question above, I think I will add a `qx.bom.client.EcmaScript#getMutationObserver` function.
Is that the right place for that information?